src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/DateTime.cs

```diff
@@ -272,7 +272,7 @@ internal sealed class WeekNumFunction : BuiltinFunction
         public override bool HasPreciseErrors => true;
 
         public WeekNumFunction()
-            : base("WeekNum", TexlStrings.AboutWeekNum, FunctionCategories.DateTime, DType.Number, 0, 1, 2, DType.DateTime, DType.Number)
+            : base("WeekNum", TexlStrings.AboutWeekNum, FunctionCategories.DateTime, DType.Number, 0, 1, 2, DType.DateTime, BuiltInEnums.StartOfWeekEnum.FormulaType._type)
```

Comment:
>BuiltInEnums.StartOfWeekEnum.FormulaType._type [](http://example.com/codeflow?start=124&length=46)

is this ok? Could this impact Power Apps?

Reply:
I spoke with PA team member, and it shouldn't. The `Weekday` function works similar to this scenario. I'm also validating this by building PA against these changes.